### ISSUE TITLE
Social Login (ADFS via SAML): blank screen

### ISSUE THREAD

**Issue by hwo-wd** [keycloakify/keycloakify#353]
Hi,

thanks for this great project, I'm just getting started and created and deployed my own theme successfully 👍.
Now, digging deeper, I configured a SAML Identity Provider[^0] which does show up on the login screen just fine:
![image](https://github.com/keycloakify/keycloakify/assets/126674117/2c67cd7e-463b-4299-9485-691bef6e32ff)

Clicking on it, I immediately get the following error[^1]:

```log
TypeError: document.forms[0] is undefined
    i SamlPostForm.tsx:13
    React 3
    x scheduler.production.min.js:13
    A scheduler.production.min.js:14
    6813 scheduler.production.min.js:14
    Webpack 12
```

`kcContext.samlPost` is populated nicely, so might the error really be related to <https://github.com/keycloakify/keycloakify/blob/69c3befb2d60bf081eeb527ed25a3bd08f2cd855/src/login/pages/SamlPostForm.tsx#LL13C60-L13C76>?
> const [htmlFormElement, setHtmlFormElement] = useState<HTMLFormElement | null>(null);

Using the stock theme is working just fine and I get authenticated flawlessly.

Thank you!
PS: I'm Using Red Hat SSO v7.6.3

[^0]: ADFS, if that should matter
[^1]: I.e., I'm not redirected to ADFS since the http post doesn't take place, because the error is faster ;-).

**Comment by garronej** [comment]
Hi @hwo-wd,  

Thank you for reaching out. Sorry about this bug.  
Could you confirm you're using the latest version of Keycloakify?   
I remember fixing somthing related to this.  

If you can confirm you're getting the error with the latest version I'll fix it ASAP

**Comment by hwo-wd** [comment]
Thanks for coming back and sorry for the missing information, but yes, I did upgrade to `7.11.11` prior to reporting.
Let me know if I can be of any help and thanks again.

**Comment by hwo-wd** [comment]
I'm so sorry for having wasted your time!
You were right that it's fixed in `7.11.11` (I did not eject said page), but I initially did all my testings with `7.11.8` and this theme was cached - even though I have disabled caching; only `/opt/sso/rh-sso-7.6/standalone/tmp/kc-gzip-cache/*` seemed to do the trick.

Again, sorry for not testing this properly and thanks again for helping!
